environment.yml

```diff
@@ -36,11 +36,19 @@ dependencies:
   - sphinxcontrib-bibtex
   - sphinx_rtd_theme
   - threadpoolctl
-  - tabulate
-  - widgetsnbextension
+  - tabulate==0.8.7
```

Comment:
Are these pins necessary? If so we need to work out why specifically and make it clear that we will undo them ASAP once the necessary conditions to remove the pins is possible (i.e. issues outlining the problem).